charts/pega/values.yaml

```diff
@@ -50,17 +50,17 @@ global:
     customerDataSchema: ""
 
   customArtifactory:
-    # If using a custom artifactory to download JDBC uri, supply the authentication details here.
-    auth:
+    # If using a custom artifactory secured with Basic or APIKey Authentication to download JDBC driver, supply the authentication details here.
+    authentication:
       # Provide either basic authentication or api key authentication details based on preferred way of authentication mechanism to custom artifactory.
```

Comment:
Provide the basic authentication credentials or the API key authentication details to satisfy your custom artifactory authentication mechanism.